lib/price-import.coffee

```diff
@@ -83,8 +83,8 @@ class PriceImport extends ProductImport
 
   _resolvePriceReferences: (price) =>
     Promise.all [
-      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.id}\"")
-      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.id}\"")
+      @_resolveReference(@client.customerGroups, 'customerGroup', price.customerGroup, "name=\"#{price.customerGroup?.groupName}\"")
+      @_resolveReference(@client.channels, 'channel', price.channel, "key=\"#{price.channel?.key}\"")
```

Comment:
I am aware of at least 2 projects that currently use it the way it is in production. Why would you go with breaking change here instead of doing it as suggested?